components/brave_wallet_ui/panel/container.tsx

```diff
@@ -484,6 +487,14 @@ function Container () {
     )
   }
 
+  if (selectedPanel === 'siweError') {
```

Comment:
can we check if the error exists instead?

Reply:
Is now checking for errors instead